src/preact-render-spy.js

```diff
@@ -264,58 +264,62 @@ class FindWrapper {
   }
 }
 
+const setHiddenProp = (object, prop, value) => {
+  Object.defineProperty(object, prop, {
+    enumerable: false,
+    configurable: true,
+    value,
+  });
+  return value;
+};
+
+class ContextRootWrapper extends Component {
+  constructor({ vdom, context }) {
+    super({ vdom, context });
+    this.state = {vdom};
+
+    // setup the re-renderer on the RenderContext
+    setHiddenProp(context, 'contextRender', vdom => {
```

Comment:
I tried it a few different ways, passing in a function that set it, this way felt better from a readability standpoint.